iceoryx_examples/icedelivery/iox_publisher_untyped_modern.cpp

```diff
@@ -29,25 +29,40 @@ static void sigHandler(int f_sig [[gnu::unused]])
     killswitch = true;
 }
 
-int main(int argc, char* argv[])
+int main()
 {
     // Register sigHandler for SIGINT
     signal(SIGINT, sigHandler);
 
-    iox::runtime::PoshRuntime::getInstance("/iox-ex-publisher-modern");
+    iox::runtime::PoshRuntime::getInstance("/iox-ex-publisher-untyped-modern");
 
     auto untypedPublisher = iox::popo::UntypedPublisher({"Odometry", "Position", "Vehicle"});
     untypedPublisher.offer();
 
     float_t ct = 0.0;
     while (!killswitch)
     {
+        ++ct;
+
+        // API Usage #1
+        //  * Loaned sample can be held until ready to publish
+        auto result = untypedPublisher.loan(128);
+        if(!result.has_error())
+        {
+            auto& sample = result.get_value();
+            new (sample.get()) Position(ct, ct, ct);
```

Comment:
Here we could explicitly mention that the placement new is needed for untyped. For the typed API, the placement new is not the normal way IMHO. And therefore returning a default constructed sample with the typed API is what I prefer